line_thresh: 0.01 #0.5
aisle_width: 0.6
